website/src/app/blog/apr-2024-update/readme.mdx

```diff
@@ -0,0 +1,245 @@
+import Image from "next/image";
+
+<Image
+  src="/images/blog/apr-2024-update/GA.png"
+  alt="Firezone 1.0 GA"
+  width={480}
+  height={320}
+  className="mx-auto rounded shadow"
+/>
+
+---
+
+## What's new:
+
+- Firezone 1.0 signups are now open!
+  [Sign up here](https://app.firezone.dev/sign_up) or
+  [request a demo](/contact/sales).
+- New [Team plan](https://app.firezone.dev/pricing) at $5 user/month.
+- [iOS](https://apps.apple.com/us/app/firezone/id6443661826) and
+  [Android](https://play.google.com/store/apps/details?id=dev.firezone.android)
+  apps are now available.
+
+### Firezone 1.0 GA
+
+After months of beta testing with our early adopters, today we're announcing
+that Firezone 1.0 is now generally available. We couldn't be more excited for
+you to try it. [Sign up](https://app.firezone.dev/sign_up) now to get started.
```

Comment:
Is it possible to force a line break after `try it.`  Only asking because the render of this paragraph just happens to make the `Sign up` link break on two different lines.

### Screenshot
<img width="716" alt="Screenshot 2024-04-09 at 9 20 03 PM" src="https://github.com/firezone/firezone/assets/2646332/2423854f-36d3-42b6-8485-7d17ca4a7cc2">

Reply:
Fixed